packages/fluent/scss/listbox/_layout.scss

```diff
@@ -0,0 +1,119 @@
+@use "../core/color-system" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-listbox--layout() {
+
+    .k-listbox {
+        width: var( --kendo-listbox-width, #{$kendo-listbox-width} );
+        height: var( --kendo-listbox-default-height, #{$kendo-listbox-default-height} );
```

Comment:
we could probably name this variable just `--kendo-listbox-height` and `$kendo-listbox-height` respectively